### ISSUE TITLE
Adding and removing breakpoints on runtime?

### ISSUE THREAD

**Issue by astrochili** [tomblind/local-lua-debugger-vscode#32]
It looks like debugger updates breakpoints only on `onDebuggerStop(msg)`.

Is it possible update breakpoints list on setBreakPointsRequest(response, args) immediately or in some timer function?

**Comment by 36yang** [comment]
I also want run time  add remove breakPoints

**Comment by tomblind** [comment]
This is challenging because lua has no built-in event loop. So, there's no straight-forward way to periodically check for incoming breakpoint changes.

A few potential options for supporting this:
- Poll for changes directly in the debug hook. This would slow down execution A LOT. It would probably be unusable.
- Provide a way to set an event loop manually. This would work for environments that have an update callback, but makes debugger setup more complicated.
- If remote debugging via sockets is added (#2), this would likely be possible via that mode.

**Comment by sneusse** [comment]
I agree with @astrochili . A `debugger.pollBreakpoints()` would be awesome! The performance penalty for polling once per frame would be neglectable. 
This could also totally be opt-in, so If you have an event loop - use it. If not don't - and get the same behaviour like now.

**Comment by tomblind** [comment]
I think this should be do-able and usefule. But I won't be able to get to it right away. It's on the roadmap though!

**Comment by MikuAuahDark** [comment]
Are there any updates on this? LOVE environment has an update loop inside `love.update` (called every game loop) where lldebugger can pull for new datas, but user must integrate it themselves.
